content/docs/concepts/traffic-management/traffic-routing/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Traffic Routing
+description: Describes traffic routing on Istio using service subsets and provides the most common uses.
+weight: 8
+keywords: [routing, request, traffic routing, requests routing, A/B testing, splitting, circuit breakers, subsets, subset]
+aliases:
+    - /docs/concepts/traffic-management/pilot
+---
+
+As described in [Pilot's architecture](../pilot), Pilot maintains an
+`Abstract Model` with the canonical representation of the services in a mesh.
+Istio's model of a service is independent of its representation in the
+underlying platform: Kubernetes, Mesos, Cloud Foundry, etc. The
+platform-specific adapters populate the `Abstract Model` representation with
+various fields from the metadata in the platform.
+
+To cover common configuration scenarios, Istio introduces the concept of a
+**service subset**, to subdivide service instances by versions, say `v1` and
+`v2`, or environment, say `staging` and `prod`. These variants are not
+necessarily different API versions: they can be iterative changes to the same
+service, deployed in different environments like prod, staging, dev, etc.
+Together with Istio's network configuration objects, you can use subsets to
+configure [routing rules](../routing-rules) that refer to service versions to
+provide additional control over the traffic between services.
+
+This concept covers the common configuration scenarios for services subsets,
+like:
+
+- [Canary rollout](#canary)
+- [Splitting traffic between versions](#splitting) for A/B testing
+- [Circuit breakers](../failures/#circuit)
+
+## Communication between services {#communication}
```

Comment:
Again, odd title - having service subsets doesn't reeeeeally allow communication between services, but the Istio config/traffic model does allow for easy communication between services that have multiple versions/subsets.  Maybe "Subsets and Virtual Services"? This whole section is a bit of a repeat of some of the stuff we've covered in the other pages, particularly around virtual services. I'd be tempted to take it out, move some of it elsewhere, or cut it down dramatically and link to the virtual services and Pilot/Envoy pages for details.